tools/build.js

```diff
@@ -253,6 +263,38 @@ Implementation-Vendor: ${PROJECT.name}
   return m;
 };
 
+function jarWebroot() {
+  JAR_INCLUDES += ` -C ${TARGET_DIR} webroot `;
+
+  var webroot = TARGET_DIR + '/webroot';
+  ensureDir(webroot);
+  copyDir('./foam3/webroot', webroot);
+  copyDir('./nanopay/src/resources/webroot', webroot);
+
+  ensureDir(webroot + '/favicon');
+  copyDir('./favicon', webroot + '/favicon');
+
+  var foambin = `foam-bin-${VERSION}.js`;
+  copyFile('./' + foambin, webroot + '/' + foambin);
+
+  copyFile('./nanopay/src/net/nanopay/ui/unauthorizedAccess/unauthorizedAccess.html', webroot + '/unauthorizedAccess.html');
```

Comment:
This shouldn't be here since build.js is generic and this is NP specific. This information should be in the POM.

Reply:
Temporarily added them here until we support before and after tasks https://github.com/kgrgreer/foam3/pull/2856 then we can move it to the appropriate project.